### PR TITLE
Remove Watson SDK templates

### PR DESCRIPTION
**Pull request by germanattanasio**
## Changes 🔥🔥🔥
 * Update dependencies to fix security issues
 * 🔥🔥🔥 Remove Watson SDK versions 🚒🚒

The idea behind this is to force starter kit authors to write the SDK code using the `IBMCloudEnv` library. That will keep things like API version and version dates with the starter kit source code rather than here where it doesn't belong.

Removing the SDK templates is a breaking change for those starters trying to use Watson services. The good thing is that the templates were not working so this won't break anything that wasn't already broken.

### CODE REVIEW

File: generators/language-python-flask/index.js [hunk 1]
```diff
@@ -278,14 +281,18 @@ module.exports = class extends Generator {
...
+		if (this.hasInstrumentation) {
+			// Remove GENERATE_HERE and GENERATE_IMPORT_HERE from SERVICES_INIT_FILE
+			let servicesInitFilePath = this.destinationPath("./server/services/" + SERVICES_INIT_FILE);
+			let indexFileContent = this.fs.read(servicesInitFilePath);
+			indexFileContent = indexFileContent.replace(GENERATE_HERE, "").replace(GENERATE_IMPORT_HERE, "");
+			this.fs.write(servicesInitFilePath, indexFileContent);
+		} else {
+			// No instrumentation was added so remove the `services` folder
+			const instrumentationFolder = this.destinationPath() + '/server/services';
+			logger.warn('No instrumentation was found, removing to ' + instrumentationFolder);
```
Comment (mamoonraja) [inline generators/language-python-flask/index.js]: `removing the` would make more sense

File: generators/language-node-express/index.js [hunk 2]
```diff
@@ -92,6 +94,7 @@ module.exports = class extends Generator {
 	}
 
 	_addInstrumentation(options){
+		this.hasInstrumentation = true;
```
Comment (mamoonraja) [inline generators/language-node-express/index.js]: we are we not adding this to `swift` and `go` too?

Comment (germanattanasio) [reply generators/language-node-express/index.js]: I don't know where `swift` and `go` are writing the instrumentation code or how that's generated for those languages

**Comment by jkingoliver** [comment]
@germanattanasio Do you intend for instrumentation code to move to `ibm-cloud-env`?

**Comment by jkingoliver** [comment]
@germanattanasio This is remove support for all our starter kits to add Watson services to an app

**Comment by christiancompton** [comment]
@germanattanasio I am declining this PR.  It is unclear what problem this is meant to address.  Please work with our team before making such drastic changes.  We are not sure what problem is trying to be solved here let alone whether this is the best path forward.  Right now this PR results significant loss of functionality.

This instrumentation code works fine on both the Apple and App Service Consoles, as well as in the `Create Project` part of the Watson console.   

> the templates were not working so this won't break anything

The templates should be working.  I have verified this in some of these templates.  If some are broken, the onus should be fixing them and not deleting large swaths of service support.  



Why is the following a useful goal?
> The idea behind this is to force starter kit authors to write the SDK code using the `IBMCloudEnv` library.

All of these pieces of instrumentation code do use IBMCloudEnv to read credentials.

> That will keep things like API version and version dates with the starter kit source code rather than here where it doesn't belong.

These snippets are starterkit source code, and are used in all starterkits where `sever.js` (etc.) is not overwritten (like in the Watson starters).  Unless version dates are provided via credentials, I don't see where this information would come from.  Additionally, the version date and the SDK being downloaded must match in terms of capabilities and backwards support.

**Comment by germanattanasio** [comment]
I'm closing this and will be opening 3 new PRs related to this

**Pull request closed without merging**